spec/models/registration_payments_spec.rb

```diff
@@ -21,5 +21,10 @@
       create(:registration_payment, :refund, registration: registration)
       expect(registration_payment.amount_available_for_refund).to eq(0)
     end
+
+    it 'does not included is_completed: false refunds' do
+      create(:registration_payment, :refund, is_completed: false, registration: registration)
+      expect(registration_payment.amount_available_for_refund).to eq(1000)
```

Comment:
This can create "false positive" situations in the UI, see my lengthy comment above

Reply:
So I'm really not sure whether this `eq(1000)` should even be the desired assumption here. You will see what I mean when you go through my example case